noisyatom/blogs/tests/test_views.py

```diff
@@ -0,0 +1,40 @@
+
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from django.utils.text import slugify
+
+
+from blogs.models import Post
+
+
+class PostModelTest(TestCase):
+
+	def create_post(self, title='Post new blog'):
+		return Post.objects.create(title=title)
+
+	def test_lis_views(self):
+		url = reverse('blogs:list')
+		response = self.client.get(url)
+		self.assertEqual(response.status_code, 200)
+
+	
+	def test_detail_views(self):
+		obj = self.create_post(title='Some new title for new test')
+		response = self.client.get(obj.get_absolute_url())
+		self.assertEqual(response.status_code, 200)
+
+
+	def test_update_views(self):
```

Comment:
this test looks wrong to me. it is checking for a 404. but it should work. in other words, you should be able to write and fetch a blog.